eng/Workarounds.targets

```diff
@@ -59,4 +59,11 @@
       <RazorReferencePath Include="@(ReferencePathWithRefAssemblies)" />
     </ItemGroup>
   </Target>
+
+  <!-- Work around https://github.com/dotnet/aspnetcore/issues/34048 -->
+  <Target Name="_RemoveDuplicateLoggingSourceGenerator" AfterTargets="ResolvePackageAssets" Condition="'$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)'">
```

Comment:
@dougbu I did a random workaround because I couldn't get `PrivateAssets` / `ExcludeAssets` to work for some reason. Could you have a look and see if there's a better way to do this?

Reply:
This looks about right b/c Logging.Abstractions is referenced transitively so often. But, I'd like to hear from @ericstj why the offending assembly is duplicated between Logging.Abstractions and our transport package.